CONTRIBUTING.md

```diff
@@ -16,15 +16,186 @@ GitBook (which we use for documentation) uses Markdown as its file format.
 * Always use inline links.
 * Do not hard-wrap prose text.  Set your text editor to soft wrapping.
 * Internal links should be absolute (starting with `/`) and link to a file ending in `.md`, not `.html`. That makes them easier to find when needed to update.
+* Each page must contain a front matter section, whether it is a single or list (`_index.md`) file. See the **Front matter** section below for more details
+
+### Front matter
```

Comment:
I think this can be brought up a header level, and thereby avoid the 4th level headers below.

Reply:
👍